app/controllers/api/functions.php

```diff
@@ -1442,22 +1852,23 @@
     ->param('functionId', '', new UID(), 'Function unique ID.', false)
     ->param('variableId', '', new UID(), 'Variable unique ID.', false)
     ->param('key', null, new Text(255), 'Variable key. Max length: 255 chars.', false)
-    ->param('value', null, new Text(8192, 0), 'Variable value. Max length: 8192 chars.', true)
+    ->param('value', null, new Text(8192), 'Variable value. Max length: 8192 chars.', true)
```

Comment:
Make sure to add the 0 min back:

```suggestion
    ->param('value', null, new Text(8192, 0), 'Variable value. Max length: 8192 chars.', true)
```